scrapy/core/scraper.py

```diff
@@ -203,10 +203,12 @@ def call_spider(
                 callback=callback, callbackKeywords=result.request.cb_kwargs
             )
         else:  # result is a Failure
-            result.request = request
+            # TODO: properly type adding this attribute to a Failure
```

Comment:
Is this TODO for this pull request or for a future one? (I’m OK with both)

Reply:
For a future one